app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -392,7 +392,7 @@ class AdministratorControlsActivityTest {
   @Test
   // TODO(#973): Fix AdministratorControlsActivityTest
   @Ignore
-  fun testAdministratorControlsFragment_loadFragment_onClickTopicUpdateOnWifiSwitch_checkSwitchRemainsChecked_onOpeningAdministratorControlsFragmentAgain() { // ktlint-disable max-line-length
+  fun testAdminCtrls_onClickTopicUpdateOnWifiSwitch_checkSwitchRemainsCheckedOnRecreate() {
```

Comment:
In general, test names should follow: ``testAction_withState_andState_hasOutcome``. Suggest renaming this to:

``testActivity_clickTopicUpdateOnWifiSwitch_reopen_switchRemainsCheckedOn``

Ditto elsewhere.

Reply:
Done!